tests/compute/test_new_kvstore.py

```diff
@@ -0,0 +1,161 @@
+import os
+import time
+
+import dgl
+import backend as F
+import unittest, pytest
+
+from numpy.testing import assert_array_equal
+
+# Create an one-part Graph
+node_map = F.tensor([0,0,0,0,0,0], F.int64)
+edge_map = F.tensor([0,0,0,0,0,0,0], F.int64)
+global_nid = F.tensor([0,1,2,3,4,5], F.int64)
+global_eid = F.tensor([0,1,2,3,4,5,6], F.int64)
+
+gpb = dgl.distributed.GraphPartitionBook(part_id=0,
+                                         num_parts=1,
+                                         node_map=node_map,
+                                         edge_map=edge_map,
+                                         global_nid=global_nid,
+                                         global_eid=global_eid)
+
+node_policy = dgl.distributed.PartitionPolicy(policy_str='node',
+                                              part_id=0,
+                                              partition_book=gpb)
+
+edge_policy = dgl.distributed.PartitionPolicy(policy_str='edge',
+                                              part_id=0,
+                                              partition_book=gpb)
+
+data_0 = F.tensor([[1.,1.],[1.,1.],[1.,1.],[1.,1.],[1.,1.],[1.,1.]], F.float32)
+data_1 = F.tensor([[2.,2.],[2.,2.],[2.,2.],[2.,2.],[2.,2.],[2.,2.],[2.,2.]], F.float32)
+data_2 = F.tensor([[0.,0.],[0.,0.],[0.,0.],[0.,0.],[0.,0.],[0.,0.]], F.float32)
+
+def init_zero_func(shape, dtype):
+    return F.zeros(shape, dtype)
+
+def udf_push(target, name, id_tensor, data_tensor):
+    target[name] = F.scatter_row(target[name], id_tensor, data_tensor*data_tensor)    
+
+def test_partition_policy():
```

Comment:
can you try partition book with multiple partitions?

Reply:
Sure, I will try that on real distributed environment.